secator/cli.py

```
@@ -593,57 +563,209 @@
 # REPORT #
 #--------#
 
 
-@cli.group(aliases=['r'])
+@cli.group(aliases=['r', 'reports'])
 def report():
 	"""Reports."""
 	pass
 
 
+def process_query(query, fields=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)